expyfun/_experiment_controller.py

```diff
@@ -933,13 +936,17 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
             A time relative to which timestamping is done. Ignored if
             timestamp==False.  If ``None``, timestamps are relative to the time
             `listen_presses` was last called.
+        type : string
+            Which key events to return. One of ``presses``, ``releases`` or
+            ``both``. (default ``presses``)
 
         Returns
         -------
         presses : list
-            If timestamp==False, returns a list of strings indicating which
-            keys were pressed. Otherwise, returns a list of tuples
-            (str, float) of keys and their timestamps.
+            Returns a list of tuples with key events. Each tuple's first value
+            will be the key pressed. If timestamp==True, the second value is
```

Comment:
surround `timestamp=True` in double-backticks to get code-formatting in the rendered docs.  I go back and forth about whether to use the double-equals in this context; single equals also makes sense if we interpret it as part of the function signature, a la `keyword=value`.  But since I can't remember which way we've been doing it, I'm going to assume that we haven't always been consistent and won't enforce one or the other here.